src/main.rs

```diff
@@ -197,41 +193,57 @@ struct Args {
     #[clap(long = "policy-timeout-ms", env = "WASMCLOUD_POLICY_TIMEOUT", value_parser = parse_duration, hide = true)]
     policy_timeout_ms: Option<Duration>,
 
-    // TODO: use and implement OCI variables
-    #[clap(long = "oci-registry", env = "OCI_REGISTRY", hide = true)]
+    /// Used in tandem with `oci_user` and `oci_password` to override credentials for a specific OCI registry.
+    #[clap(
+        long = "oci-registry",
+        env = "OCI_REGISTRY",
+        requires = "oci_user",
+        requires = "oci_password"
+    )]
     oci_registry: Option<String>,
+    /// Username for the OCI registry specified by `oci_registry`.
     #[clap(
         long = "oci-user",
         env = "OCI_REGISTRY_USER",
-        requires = "oci_password",
-        hide = true
+        requires = "oci_registry",
+        requires = "oci_password"
     )]
     oci_user: Option<String>,
+    /// Password for the OCI registry specified by `oci_registry`.
     #[clap(
         long = "oci-password",
         env = "OCI_REGISTRY_PASSWORD",
-        requires = "oci_user",
-        hide = true
+        requires = "oci_registry",
+        requires = "oci_user"
     )]
     oci_password: Option<String>,
 }
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let Args {
+        log_level,
         nats_host,
         nats_port,
         lattice_prefix,
         host_seed,
         cluster_seed,
+        cluster_issuers,
+        provider_shutdown_delay,
+        allow_latest,
+        allowed_insecure,
+        oci_registry,
+        oci_user,
+        oci_password,
+        js_domain,
         ..
```

Comment:
nit; if all the args are used now, can we remove the `..` to force a compile-time error when an unused arg is added?